skimage/future/__init__.py

```diff
@@ -5,6 +5,6 @@
 production code that will depend on updated skimage versions.
 """
 
-from . import graph
+from . import graph, objdetect
 
-__all__ = ['graph']
+__all__ = ['graph', 'objdetect']
```

Comment:
Definitely start with `detect`. I'd be fine with `detect`, `detect_object`, and probably others.